cumulusci/cumulusci.yml

```diff
@@ -98,6 +98,13 @@ tasks:
         options:
             command: 'force:source:push'
         group: Salesforce DX
+    ensure_record_types:
```

Comment:
🤔 : Why did I think this was already a task?

Reply:
It was, but it wasn't exposed in a defined task - just per-project.